src/ddmd/escape.d

```diff
@@ -665,7 +669,7 @@ private bool checkReturnEscapeImpl(Scope* sc, Expression e, bool refs, bool gag)
 
     foreach (Expression ee; er.byexp)
     {
-        //printf("byexp %s\n", ee.toChars());
+        if (log) printf("byexp %s\n", ee.toChars());
```

Comment:
Same here. More than half of the diff in this file is logging / debugging change. In my opinion it shouldn't be there to begin with, but if you want it here, it should at least be in its own commit.

Reply:
It's trivial, and does not obfuscate this PR. I often leave in debugging aids when I develop PRs, they are not completely unrelated.

Reply:
Part of the issue (for me) is the looooong delays in getting even the most trivial PRs pulled. These debug aids are very helpful to me.

Reply:
> Part of the issue (for me) is the looooong delays in getting even the most trivial PRs pulled. 

I would happily merge the `scope` related P.R. if I had access right. This is an area where I can pretend to know what I'm doing ;)

Reply:
I added pull privileges to you, but keep in mind that Leandro and Dicebot both have pull privileges, too.

Reply:
> I added pull privileges to you, but keep in mind that Leandro and Dicebot both have pull privileges, too.

I know they have pull access, but AFAIK they only care about sociomantic-related stuff nowadays.
`scope` is of no interest for us ATM (we're using 2.070 / 2.071), so the time used to work on it is part of our [open source initiative](https://github.com/sociomantic-tsunami).

I am still not seeing the option to pull tho :confused: